src/resolvers/registries/npm-resolver.js

```diff
@@ -107,8 +107,19 @@ export default class NpmResolver extends RegistryResolver {
 
     const files = await this.config.getCache('cachedPackages', async (): Promise<Array<string>> => {
       const files = await fs.readdir(cacheFolder);
-      const validFiles = [];
+      const scoped_files = {};
+      for (const scope of files) {
+        // no hidden files
+        if (scope[0] === '.') {
```

Comment:
I think this check should be extended to only accept things starting with an `@` since that denotes a scope.